packages/@expo/json-file/src/__tests__/JsonFile-test.ts

```diff
@@ -80,7 +80,8 @@ describe('async', () => {
     vol.fromJSON({ [testFilename]: loadFixture('syntax-error.json') });
 
     await expect(JsonFile.readAsync(testFilename)).rejects.toThrowError(
-      /Cause: SyntaxError: Unexpected string in JSON at position 602/
+      // based on node version you may get different error messages
```

Comment:
Might be nice to put a link or info to what changed, e.g. "Since Node 23+ errors JSON parsing errors mention the invalid character"